homework_9/homework_9.py

```diff
@@ -0,0 +1,11 @@
+
+def fibonacci(n):
+    n = input("Enter a number of fibonacci element: ")
+    n = int(n)
+    a, b = 1, 1
+    for i in range(n):
+        yield a
+        a, b = b, a + b
+
+data = list(fibonacci(10))
+print(data)
```

Comment:
The task was to print `n`th number of sequence. For example, if user entered `7` your program should pring `13`.
In your case, after entering `7` your program prints `[1, 1, 2, 3, 5, 8, 13]`

Please, fix your solution